Babylon/groups/api/groups/workspace/commands/delete.py

```diff
@@ -78,10 +82,14 @@ def delete(
             logger.error("Can not get Workspace definition, please check your file")
             return
 
-        workspace_id = converted_workspace_content["id"] or converted_workspace_content["workspace_id"]
-        if not workspace_id:
-            logger.error(f"Could not found workspace id in {workspace_file}.")
-            return
+        try:
+            workspace_id = converted_workspace_content["id"]
+        except KeyError:
+            try:
+                workspace_id = converted_workspace_content["workspace_id"]
+            except KeyError:
+                logger.error("Can not get solution id, please check your file")
+                return
```

Comment:
Here is another way to do this, but yours is fine
```suggestion
    workspace_id = converted_workspace_content.get("id") or converted_workspace_content.get("workspace_id")
    if not workspace_id:
        logger.error("Can not get solution id, please check your file")
        return    
```